tutorials/fundamentals/primitives.md

```diff
@@ -0,0 +1,56 @@
+Javascript has seven primitive data types:
+
+- string
+- number
+- bigint
+- boolean
+- undefined
+- symbol
+- null
+
+What makes these primitives is that they are not objects,
+they do not have any methods, and they are immutable,
+meaning that they cannot be altered in any way.
+If you wanted to mutate a primitive, you would need
+to do a reassignment of the variable.
+
+## String
+
+It is important to note that strings can be either a primitive, string literal,
```

Comment:
Instead of explaining each individual primitive type, let's add some examples of using `typeof` to distinguish between different types.